src/main/php/io/modelcontextprotocol/Capabilities.class.php

```diff
@@ -28,6 +28,16 @@ public static function client(bool $sampling= true, bool $roots= true, ?array $e
     ]);
   }
 
+  /** Creates server capabilities */
+  public static function server() {
+    return new self([
+      'logging'   => (object)[],
+      'prompts'   => (object)[],
+      'resources' => (object)[],
+      'tools'     => ['listChanged' => true],
```

Comment:
The listChanged here should not appear, we don't actually support notifications for changed tools.